site/en/blog/help-css-nesting/index.md

```diff
@@ -0,0 +1,776 @@
+---
+layout: layouts/blog-post.njk
+title: Help CSS nesting pick syntax
+description: >
+  Two competing syntaxes need your help in determining which should be championed through to a spec candidate.
+subhead: >
+  Two competing syntaxes need your help in determining which should be championed through to a spec candidate.
+date: 2022-07-14
+authors:
+  - argyle
+  - miriamsuzanne
+tags:
+  - css
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/ZrMGmOcY6AH5J2tAblyO.jpg
+alt: >
+  A bird nest with a couple of blue eggs inside, surrounded by lush greenery.
+---
+
+CSS nesting is a convenience syntax addition that allows CSS to be added inside
+of a ruleset. If you've used
+[SCSS](https://sass-lang.com/guide#:~:text=throughout%20the%C2%A0site.-,Nesting,-When%20writing%20HTML),
+[Less](https://lesscss.org/features/#features-overview-feature-nested-rules) or
+[Stylus](https://stylus-lang.com/docs/selectors.html#partial-reference), then
+you've most certainly seen a few flavors of this:
+
+```scss
+.nesting {
+  color: hotpink;
+
+  > .is {
+    color: rebeccapurple;
+
+    > .awesome {
+      color: deeppink;
+    }
+  }
+}
+```
+
+Which after being compiled to regular CSS by the preprocessor, turns into
+regular CSS like this:
+
+```css
+.nesting {
+  color: hotpink;
+}
+
+.nesting > .is {
+  color: rebeccapurple;
+}
+
+.nesting > .is > .awesome {
+  color: deeppink;
+}
+```
+
+An official CSS version of this syntax is being strongly considered and we have
+a split in preference that we'd like to employ the help of the community to
+break the tie. The rest of this post will be introducing the syntax options so
+you can feel informed to take a survey at the end.
+
+## Overview of what we're comparing
```

Comment:
Would it help to explain why the syntax above (used by pre-processors) isn't an option?

Reply:
sounds reasonable to me. do you happen to have a link we can provide that goes over it? or maybe want to offer a blurb?

Reply:
I documented some of it here: https://github.com/sass/sass/issues/3030

Not sure there's any reason to link there, but you could borrow some of the language.

Reply:
added a section about it. thoughts? it's a tough one to touch on, hope i did ok